_studio/mfx_lib/encode_hw/vp9/src/mfx_vp9_encode_hw.cpp

```diff
@@ -107,16 +104,13 @@ mfxStatus MFXVideoENCODEVP9_HW::QueryIOSurf(mfxCoreInterface *core, mfxVideoPara
 
     VP9MfxVideoParam toValidate = *par;
 
-    mfxCoreInterface _core = *core;
-
     // get HW caps from driver
     ENCODE_CAPS_VP9 caps = {};
-    mfxPlatform platform = {};
-    MFX_CHECK(MFX_ERR_NONE == QueryCapsAndPlatform(&_core, caps, platform, GetGuid(toValidate), par->mfx.FrameInfo.Width, par->mfx.FrameInfo.Height), MFX_ERR_UNSUPPORTED);
+    MFX_CHECK(MFX_ERR_NONE == QueryCaps(core, caps, GetGuid(toValidate), par->mfx.FrameInfo.Width, par->mfx.FrameInfo.Height), MFX_ERR_UNSUPPORTED);
```

Comment:
I suggest to get the function call out of macro. During the macro expansion at debug current approach will cause function to be called twice

Reply:
Good point! Despite it is not reproduced for this particular case, I'd better move the function out of the macros to prevent possible issue in future.

Reply:
Not addressed.